declarations/WebpackOptions.d.ts

```diff
@@ -2893,7 +2893,7 @@ export interface JavascriptParserOptions {
 	/**
 	 * Emit errors instead of warnings when imported names don't exist in imported module.
 	 */
-	strictExportPresence?: boolean;
+	strictExportPresence?: false | true | "error" | "warn" | "ignore";
```

Comment:
It is breaking change

Reply:
I believe the way I implemented this is non-breaking. Prior to this, only Boolean is accepted, now it's either Boolean or strong representation. My changes in normalization.js continue to accept booleans and convert them to the relevant string representation (just as before). Any prior setting of this value (as either true or false) will continue to work as expected, as the confit will be changed to error or warn during normalization. 

I can even revert all test changes and still see tests continue to pass because of normalization.